Dockerfile

```diff
@@ -19,48 +19,24 @@ ARG ALPINE_VERSION=3.12
 FROM --platform=$BUILDPLATFORM alpine:${ALPINE_VERSION} AS git
 RUN apk add --no-cache git
 
-# xgo is a helper for golang cross-compilation
-FROM --platform=$BUILDPLATFORM tonistiigi/xx:golang@sha256:6f7d999551dd471b58f70716754290495690efa8421e0a1fcf18eb11d0c0a537 AS xgo
+# xx is a helper for cross-compilation
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:golang@sha256:810dc54d5144f133a218e88e319184bf8b9ce01d37d46ddb37573e90decd9eef AS xx
+
+FROM --platform=$BUILDPLATFORM golang:1.13-alpine AS gostable
+FROM --platform=$BUILDPLATFORM golang:1.16-alpine AS golatest
+
+FROM gostable AS go-linux
+FROM golatest AS go-darwin
+FROM golatest AS go-windows-amd64
+FROM golatest AS go-windows-386
```

Comment:
Do we want to support windows 386?

Reply:
I only left amd64&arm64 in the CI. These are just to cover all possible options. Once we don't need to switch between go versions this can be all removed.